implementations/go/README.md

```diff
@@ -1,5 +1,10 @@
 # Soccergist - Go
 
+Install go-lang on your system http://golang.org
+
+
```

Comment:
Also add the command to run that app. Should be `go run *.go` right?

Reply:
Alright... will add it.. its "go run main.go". But will add it to the readme.